src/sage/rings/finite_rings/finite_field_constructor.py

```diff
@@ -207,7 +207,7 @@ class FiniteFieldFactory(UniqueFactory):
 
     INPUT:
 
-    - ``order`` -- a prime power
+    - ``order`` -- a prime power `p^n`, or a pair `(p,n)`
 
     - ``name`` -- string, optional.  Note that there can be a
```

Comment:
```suggestion
    - ``name`` -- string or integer, optional. For a string,
      ``name`` is the name of the generator over the prime
      field. For convenience, in ``name`` is an integer, then
      ``GF(p, n)`` is equivalent to ``GF((p, n))`` (see examples below).
      Note that there can be a substantial speed penalty
      (in creating extension fields) when omitting the variable name,
      since doing so triggers the computation of pseudo-Conway
      polynomials in order to define a coherent lattice of extensions
       of the prime field.  The speed penalty grows with the size of
       extension degree and with the number of factors of the
       extension degree.
```

I think this is a bit more clear (and fixes the incorrect type description).

I'm making this on GitHub which doesn't show me if I'm exceeding the line lengths, you may have to fix the formatting if the linter complains.

Reply:
The linter is pretty relaxed nowadays (the line limit has been bumped to something like 120? Not sure), this is just very old code that sticks to ≈ 80 character line limit. Either way, doesn't matter.